tests/e2e_tests/conftest.py

```
@@ -94,11 +94,23 @@ def test_something(qdrant_image):
         else:
             print(f"Building Docker image {image_tag}...")
 
-        # Build image using docker buildx
+        cache_dir = Path.home() / ".cache" / "qdrant-e2e-buildx"
+        builder_name = "qdrant-e2e-builder"
+        if subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)